apps/src/lab2/views/components/layout/variables.scss

```diff
@@ -2,3 +2,10 @@
 
 // Default border style
 $default-border: 1px solid var(--borders-neutral-primary);
+
+/** z-indices */
```

Comment:
FYI @kelbyhawn @molly-moen - I started adding some z-index constants here (thanks Kelby for creating this file!), in case its relevant as you're looking at cross-Lab2 UI. I only added values relevant to this PR here to avoid unintentional side effects, but if this pattern is useful, we can cleanup and migrate more as we come across them.